src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,67 @@
+import clsx from "clsx";
+import { Card, CardVariant } from "components/Card/Card";
+import { ArrowDropdownDownIcon } from "components/Icons/ArrowDropdownDownIcon";
+// import { Icon } from "components/Icon/Icon";
```

Comment:
Jade said she always want to have the dropdown icon when accordion is closed and the icon that points up when the dropdown is open